Gordon360/Services/LostAndFoundService.cs

```diff
@@ -61,6 +61,23 @@ public async Task UpdateMissingItemReportAsync(int id, MissingItemReportViewMode
             }
         }
 
+        /// <param name="id">The id</param>
+        public async Task UpdateReportStatusAsync(int id, MissingItemReportViewModel reportDetails)
```

Comment:
Again, you only use the status out of the MissingItemDetails object, so no point in requiring the frontend to pass the whole object, just pass the data you need.